server/src/main/java/io/spine/server/entity/Transaction.java

```diff
@@ -494,6 +495,32 @@ public void setDeleted(boolean deleted) {
                                        .build();
     }
 
+    /**
+     * Advances the transaction version.
```

Comment:
I think we should be more precise here, as transactions do not have versions themselves.